android-lib/src/main/java/com/kinvey/android/store/DataStore.java

```diff
@@ -542,13 +546,29 @@ public void save(@NonNull T entity, @NonNull KinveyClientCallback<T> callback)
      * @param callback KinveyClientCallback<List<T>>
      */
     public void save(@NonNull List<T> entities, @NonNull KinveyClientCallback<List<T>> callback)  {
+         if (KINVEY_API_VERSION_5.equals(kinveyApiVersion)) {
+             saveBatch(entities, callback);
+         } else {
+             saveV4(entities, callback);
+         }
+    }
+
+    private void saveV4(@NonNull List<T> entities, @NonNull KinveyClientCallback<List<T>> callback)  {
         Preconditions.checkNotNull(client, "client must not be null");
         Preconditions.checkArgument(client.isInitialize(), "client must be initialized.");
         Preconditions.checkNotNull(entities, "Entity cannot be null.");
         Logger.INFO("Calling DataStore#save(listObjects)");
         new SaveListRequest(entities, callback).execute();
     }
 
+    private void saveBatch(@NonNull List<T> entities, @NonNull KinveyClientCallback<List<T>> callback)  {
+        Preconditions.checkNotNull(client, "client must not be null");
+        Preconditions.checkArgument(client.isInitialize(), "client must be initialized.");
+        Preconditions.checkNotNull(entities, "Entity cannot be null.");
+        Logger.INFO("Calling DataStore#saveBatch(listObjects)");
```

Comment:
Need another Precondition here if the number of items in the list of entities is greater than 100.

Reply:
I've added new precondition `checkPositionIndex` that will stop execution if user will try to send more than 100 items per request.